apps/judicial-system/web/src/components/SharedCasesLayout/SharedCasesLayout.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC, PropsWithChildren } from 'react'
+
+import { Box } from '@island.is/island-ui/core'
+
+import SectionHeading from '../SectionHeading/SectionHeading'
+import * as styles from './SharedCasesLayout.css'
+
+interface Props {
+  title: string
+}
+
+const SharedCasesLayout: FC<PropsWithChildren<Props>> = ({
```

Comment:
Please consider renaiming the component to something like `TableGroup` as its only purpose is to group together cards/links to tables. It could be used for something other than cases, for example user admin.